.travis.yml

```diff
@@ -64,13 +64,16 @@ jobs:
     - stage: validate
       script: mvn -T 4 --quiet modernizer:modernizer
       name: "Modernizer Legacy API Analysis"
+    - stage: validate
+      script: mvn -T 4 --quiet apache-rat:check
+      name: "Release Audit Tool (RAT) Analysis"
       ######################################################
     - stage: test
       script: mvn clean test -T 4 -P skipLegalFiles,skipSass -T 4 --quiet -Dinvoker.streamLogs=true -Denforcer.skip=true -Dmodernizer.skip=true
       name: "Unit Tests"
       ######################################################
     - stage: fit
-      script: travis_wait 30 mvn -f fit/core-reference/pom.xml verify -P skipLegalFiles,skipSass -T 4 -Dinvoker.streamLogs=true -Denforcer.skip=true -Dmodernizer.skip=true -Djacoco.skip=true
+      script: travis_wait 45 mvn -f fit/core-reference/pom.xml verify -P skipLegalFiles,skipSass -T 4 -Dinvoker.streamLogs=true -Denforcer.skip=true -Dmodernizer.skip=true -Djacoco.skip=true -Drat.skip=true
```

Comment:
here you can add `-P full-it,all` to trigger Admin Console and Enduser UI integration tests too

Reply:
Can those tests be run independently in a separate job perhaps to save time, with another profile perhaps? 

Or maybe we can add two more jobs with these?

- travis_wait 45 mvn -f fit/enduser-reference/pom.xml ...
- travis_wait 45 mvn -f fit/console-reference/pom.xml ...

..giving each 45 minutes. That might work, time-wise?

Reply:
The tests are still in `fit/core-reference`, but excluded by default.

So it should be something like as:
```yaml
script: travis_wait 45 mvn -f fit/core-reference/pom.xml verify -P full-it,all,skipLegalFiles,skipSass -T 4 -Dinvoker.streamLogs=true -Denforcer.skip=true -Dmodernizer.skip=true -Djacoco.skip=true -Drat.skip=true
```

Reply:
Got it, will do.

I see some IT tests inside the enduser-reference module for OIDC and SAML2SP. Should we enable and run those in the CI? Not quite sure how they are executed now.

Reply:
> I see some IT tests inside the enduser-reference module for OIDC and SAML2SP. Should we enable and run those in the CI? Not quite sure how they are executed now.

They require Docker, guess we'll handle them later, no?

Reply:
Absolutely. 

I guess [this page](https://syncope.apache.org/building#fit.2Fconsole-reference) should be slightly updated to make the docker requirement clearer. Today's reading suggests that docker is only required for DB tests and console/enduser tests can just run on their own...or am I reading that wrong?

Reply:
That page mentions `fit/enduser-reference` and `fit/console-reference` only for `debug` and `hotswap` profiles, no tests.

Moreover, the profile `sso-it` in `fit/enduser-reference` is only available in the `master` branch at the moment.

Reply:
I'll take note in the follow-up JIRA when I rework ITs based on Docker. Thank you!